src/AutoLazy.Fody/LazyVisitor.cs

```diff
@@ -8,25 +8,34 @@ internal class LazyVisitor : IMethodVisitor, IPropertyVisitor
     {
         public void Visit(MethodDefinition method, VisitorContext context)
         {
-            if (IsLazy(method))
+            var lazyAttribute = GetLazyAttribute(method);
+            if (lazyAttribute != null)
```

Comment:
If the property has `[Lazy]` on it but the method does not it looks like the method won't get instrumented correctly now. This is because the method instrumenter now requires the attribute be present to instrument.